### DISCUSSION TITLE
Regarding Pull Request and Inclusion in PrestaShop v9.0

### DISCUSSION THREAD

**Discussion by OrionCodeDev** [PrestaShop/PrestaShop#36449] [Q&A] [answered]
A few months ago, I submitted a pull request to include the option for exporting products in the new product page version 2 for PrestaShop version 8.1.X. I was pleased to contribute to the platform and hoped that this feature would be beneficial to other users as well.

pull link - https://github.com/PrestaShop/PrestaShop/pull/35488

However, I recently noticed that the newly released version 9.0 does not include this change. Could you please advise on the appropriate process to ensure that this feature is also integrated into the latest version? Is there a specific channel or repository where I can report this, or should I submit a new pull request for version 9.0?

Thank you for your assistance and for your continuous efforts in improving PrestaShop.

**Comment by kpodemski** [comment, marked as answer]
Hello @ChronoBrake 

Alpha 1 Build didn't have this change because the merge commit was created later. Here's the PR that includes your change into PrestaShop 9:
https://github.com/PrestaShop/PrestaShop/pull/36394

So, don't worry, your change will be available in the final version of PrestaShop 9 :)
